hedera-node/hedera-app/src/testFixtures/java/module-info.java

```diff
@@ -23,7 +23,6 @@
     requires com.swirlds.config.extensions.test.fixtures;
     requires com.swirlds.platform.core.test.fixtures;
     requires com.swirlds.state.impl.test.fixtures;
-    requires com.swirlds.state.impl;
```

Comment:
Not required as direct dependency. Detected after [dependency analysis plugin update](https://github.com/hiero-ledger/hiero-gradle-conventions/releases/tag/v0.3.9) as part of this PR.

Reply:
@hiero-ledger/hiero-consensus-node-execution-codeowners can someone please approve this change?

Reply:
🎉